docs/yearmonth.md

```diff
@@ -377,21 +377,56 @@ Usage example:
 
 <!-- prettier-ignore-start -->
 ```javascript
-ym = Temporal.YearMonth.from('2019-06');
-other = Temporal.YearMonth.from('2006-08');
-ym.since(other);                            // => P12Y10M
-ym.since(other, { largestUnit: 'months' }); // => P154M
-other.since(ym, { largestUnit: 'months' }); // => -P154M
+ym = Temporal.YearMonth.from('2006-08');
+other = Temporal.YearMonth.from('2019-06');
+ym.until(other);                            // => P12Y10M
+ym.until(other, { largestUnit: 'months' }); // => P154M
+other.until(ym, { largestUnit: 'months' }); // => -P154M
 
 // If you really need to calculate the difference between two YearMonths
 // in days, you can eliminate the ambiguity by explicitly choosing the
 // day of the month (and if applicable, the time of that day) from which
 // you want to reckon the difference. For example, using the first of
 // the month to calculate a number of days:
-ym.toDateOnDay(1).since(other.toDateOnDay(1), { largestUnit: 'days' }); // => P4687D
+ym.toDateOnDay(1).until(other.toDateOnDay(1), { largestUnit: 'days' }); // => P4687D
 ```
 <!-- prettier-ignore-end -->
 
+### yearMonth.**since**(_other_: Temporal.YearMonth | object | string, _options_?: object) : Temporal.Duration
+
+**Parameters:**
+
+- `other` (`Temporal.YearMonth` or value convertible to one): Another month since when to compute the difference.
+- `options` (optional object): An object with properties representing options for the operation.
+  The following options are recognized:
+  - `largestUnit` (string): The largest unit of time to allow in the resulting `Temporal.Duration` object.
+    Valid values are `'auto'`, `'years'` and `'months'`.
+    The default is `'auto'`.
+  - `smallestUnit` (string): The smallest unit of time to round to in the resulting `Temporal.Duration` object.
+    Valid values are `'years'` and `'months'`.
+    The default is `'months'`, i.e., no rounding.
+  - `roundingIncrement` (number): The granularity to round to, of the unit given by `smallestUnit`.
+    The default is 1.
+  - `roundingMode` (string): How to handle the remainder, if rounding.
+    Valid values are `'nearest'`, `'ceil'`, `'trunc'`, and `'floor'`.
+    The default is `'nearest'`.
+
+**Returns:** a `Temporal.Duration` representing the elapsed time before `yearMonth` and since `other`.
+
+This method computes the difference between the two months represented by `yearMonth` and `other`, optionally rounds it, and returns it as a `Temporal.Duration` object.
+If `other` is later than `yearMonth` then the resulting duration will be negative.
+
+This method does the same thing as the `Temporal.YearMonth.prototype.until()` method, but reversed, and rounding takes place relative to `yearMonth` as an ending point instead of a starting point.
+If no rounding takes place, then the outcome of `ym1.since(ym2)` is the same as `ym1.until(ym2).negated()`.
```

Comment:
I think that the same reversibility issue as noted above applies here too, although I'm not 100% sure. It might only happen in calendars with variable numbers of months in each year, and with a starting or ending point inside a leap month.